presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```
@@ -638,6 +640,9 @@ public boolean trySetBytes(long bytes)
         @Override
         public void close()
         {
+            if (!closable) {
+                throw new IllegalStateException("Called close on unclosable local memory context");
```

nit: I feel like we should throw an unsupported operation exception.